pkg/apis/settings/injectable.go

```diff
@@ -0,0 +1,27 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package settings
```

Comment:
imo this belongs outside of the APIs package.

Reply:
Did you have a thought about where else this would be sensible to go? Settings IMO are definitely part of our API, even if not directly through a CR. As such, the settings definitely belong inside of the `apis` package. To reduce the dependency tree in the `apis` package, it seems sensible that these would be part of the same package.

Reply:
operator/injection?